docs/plugin-transform-typescript.md

```diff
@@ -198,6 +206,8 @@ equivalents in Babel can be enabled by some configuration options or plugins.
   ```
 - `--importHelpers`
   This is the equivalent of the `@babel/plugin-transform-runtime` package.
+- `---importsNotUsedAsValues`
+  You can the `onlyRemoveTypeImports` option to replicate this behavior. `onlyRemoveTypeImports: true` is equivalent to `importsNotUsedAsValues: preserve`, while `onlyRemoveTypeImports: false` is equivalent to `importsNotUsedAsValues: remove`. There is no equivalent for `importsNotUsedAsValues: error`.
```

Comment:
Does this seem correct/clear?

Reply:
Maybe we can't mention that `onlyRemoveTypeImports: false` isn't 100% accurate because we don't do any type analysis, and link to https://github.com/babel/babel/issues/8361 as an example? Also, it's the reason why we don't support `importsNotUsedAsValues: error`.

Reply:
Do you mind clarifying what you mean?

Reply:
Sure!

Consider this example:
```ts
// dep.ts

export type T = string;
```
```ts
// main.ts

import { T } from "./dep";
export { T };
```

`onlyRemoveTypeImports: false` leave `main.ts` as-is, because we don't have the type information necessary to understand that `T` is a type.
However, TypeScript's `importsNotUsedAsValues: remove` transforms it to an empty file because it knows that `T` is a type.